sources/api/simplefin.py

```
@@ -0,0 +1,117 @@
+
+class SimplefinStatement(OnlineCardStatement):
+    def __init__(self):
+        super().__init__(type="simplefin")
+
+    def load_data(self) -> None:
+        setup_token = self.config.simplefin_setup_token
+        if not setup_token:
+            raise ValueError(
+                "SIMPLEFIN_SETUP_TOKEN not set. Get one at "
+                f"{BRIDGE_CREATE_URL} and add it to .env"
+            )
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)